pnpm-lock.yaml

```diff
@@ -9,7 +9,7 @@ overrides:
   semver-max: file:./npm/private/test/vendored/semver-max
   is-odd: file:./npm/private/test/vendored/is-odd
 
-packageExtensionsChecksum: e978e3e510efb71c3833e1915611f5d9
```

Comment:
Why did this change? 🤔

Reply:
Good question 🤔

Reply:
Unwound it and the delta went away. No idea what happened. 🤷